api/controllers/routes/agency.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const app = express();
+const requestUtils = require("../../utils/request");
+const queryUtils = require("../../utils/query");
+
+const env = process.env.NODE_ENV || 'development';
+
+const config = require("../../../config")[env];
+const host = config.host;
+const agencyResourceId = config.agency_resource;
+const agencyServiceResourceId = config.agency_service_resource;
+const serviceTaxonomyResourceId = config.service_taxonomy_resource;
+
+const uri = `${host}/api/3/action/datastore_search_sql?sql=SELECT * from "${agencyResourceId}" agency `;
```

Comment:
This especially since it looks pretty standard. Another thing, can we be consistent with the SQL statement? I mean SELECT * FROM or select * from versus SELECT * from.

Reply:
I can make this a constant but then it will be a series of variables with no context. I think readability might suffer:
```
const uri = `${host}${query}${agencyResourceId}`....
```